<!DOCTYPE html>
<html lang="en">
<head>
    <title>Software | Mesirov Lab</title>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
    <!-- IE6-10 -->
    <link rel="shortcut icon" href="../img/meslab.ico">
    <!-- Everybody else -->
    <link rel="icon" href="../img/meslab.ico">
    <link type="text/css" rel="stylesheet" href="../css/styles.css">
    
    <!-- Global site tag (gtag.js) - Google Analytics -->
    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-130775773-1"></script>
    <script>
        window.dataLayer = window.dataLayer || [];
        function gtag() { dataLayer.push(arguments); }
        gtag('js', new Date());

        gtag('config', 'UA-130775773-1', {
            'linker': {
                'accept_incoming': true
            }
        });
    </script>
    <script>
        $("#myScrollspy ul li a[href^='#']").on('click', function (e) {

            // prevent default anchor click behavior
            e.preventDefault();

            // store hash
            var hash = this.hash;

            // animate
            $('html, body').animate({
                scrollTop: $(hash).offset().top
            }, 300, function () {

                // when done, add hash to url
                // (default click behaviour)
                window.location.hash = hash;
            });

        });
    </script>

</head>
<body >
    <nav class="navbar navbar-fixed-top">
        <a class="navbar-brand " href="..">
            <p style="color: #172A49 !important;">Mesirov Lab</p>
        </a>
        <div class="navbar-header">
            <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#myNavbar">
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
            </button>
        </div>
        <div class="collapse navbar-collapse" id="myNavbar">
            <ul class="nav navbar-nav" id="navigation">
                <span id="media-space" style="height: 80px"></span>
                <li><a href="/members/">OUR TEAM</a></li>
                <li><a href="/pubs/">PUBLICATIONS</a></li>
                <li><a href="/software/">SOFTWARE/DATA RESOURCES</a></li>
                <li><a href="/teaching/">TEACHING</a></li>
                <li><a href="/about/">CONTACT</a></li>
                <li><a href="/opportunities/">JOIN US</a></li>
            </ul>
            <a class="navbar-brand navbar-right" href=".."
                style="background: white; margin: auto; padding-right: 50px;padding-left: 50px; padding-top: 35px; padding-bottom: 65px; margin-right: 10px; float:right"
                id="ucsd-logo">
                <img src="../img/UCSD_logo.png" width="150px" style="padding:0px; margin:auto;">
            </a>
        </div>
    </nav>
    
    <div class="row" style="margin-top:25px;">
        <div id="software-panels" class="col-sm-10" style="background: rgba(255, 255, 255, 0.7); padding: 50px;" >
            
            <div id="title">
                <h1>SOFTWARE AND DATA RESOURCES</h1>
                <br>
            </div>
            
            <div style="background: white;padding: 30px;" class="row" id="section1">
                <div class="col-sm-6">
                    <h2><a target="_blank" href="https://genepattern.broadinstitute.org">                            <img src="../img/software/genepattern.png" width="50px;"> GenePattern</a></h2>
                    <hr>
                    <p>We have developed a software package called GenePattern, which provides a comprehensive
                        environment that can support a broad community of users at all levels of computational
                        experience and sophistication, access to a repository of analytic and visualization tools and
                        easy creation of complex analytic methods from them and the rapid development and dissemination
                        of new methods. Perhaps the most important feature of GenePattern is that it supports a
                        mechanism to guarantee the capture and independent replication of published computational
                        methods and in silico results.</p>
                </div>
                <div style="text-align: center;" class="col-sm-6">
                    <img src="../img/genepattern-soft2.png" alt="genepattern" width="100%">
                </div>
            </div>

            <div style="background: white;padding: 30px;" class="row" id="section2">
                <div style="text-align: center;" class="col-sm-6">
                    <img src="../img/gpnb-soft.png" alt="GP-Notebooks" width="100%">
                </div>
                <div id="section2" class="col-sm-6">
                    <h2><a target="_blank" href="https://notebook.genepattern.org/"><img src="../img/software/genepattern.png" width="50px;"> GenePattern Notebook</a></h2>
                    <hr>
                    <p>Interactive analysis notebook environments promise to streamline genomics research
                        through interleaving text, multimedia, and executable code into unified, sharable,
                        reproducible “research narratives.” However, current notebook systems require
                        programming knowledge, limiting their wider adoption by the research community. We have
                        developed the GenePattern Notebook environment, to our knowledge the first system to
                        integrate the dynamic capabilities of notebook systems with an investigator-focused,
                        easy-to-use interface that provides access to hundreds of genomic tools without the
                        need to write code.</p>
                </div>
            </div>

            <div style="background: white; padding: 30px;" class="row" id="section3">
                <div id="section2" class="col-sm-6">
                    <h2><a target="_blank" href="https://igv.org"><img src="../img/software/11527111.png" width="50px;"> IGV</a>
                    </h2>
                    <hr>
                    <p>The Integrative Genomics Viewer (IGV) is a high-performance visualization tool for
                    interactive exploration of large, integrated genomic datasets. It supports a wide
                    variety of data types, including array-based and next-generation sequence data, and
                    genomic annotations.</p>
                </div>
                <div style="text-align: center;" class="col-sm-6">
                    <img src="../img/igv-soft.png" alt="igv" width="100%">(1)
                </div>
            </div>

            <div style="background: white; padding: 30px;" class="row" id="section4">
                <div style="text-align: center;" class="col-sm-6">
                    <img src="../img/igv-js-soft.png" alt="igv.js" width="100%">
                </div>
                <div id="section2" class="col-sm-6">
                    <h2><a target="_blank" href="https://github.com/igvteam/igv.js"><img src="../img/software/11527111.png" width="50px;"> igv.js</a>
                    </h2>
                    <hr>
                    <p>igv.js is an embeddable interactive genome visualization component written in JavaScript and CSS.
                    It is based on the Integrative Genomics Viewer(IGV), and developed by the same team.</p>
                </div>
                
            </div>

            <div style="background: white; padding: 30px;" class="row" id="section5">
                <div id="section2" class="col-sm-6">
                    <h2><a target="_blank" href="https://igv.org/doc/juiceboxjs.html"><img src="../img/software/11527111.png" width="50px;"> Juicebox.js</a>
                    </h2>
                    <hr>
                    <p>Juicebox.js is an interactive contact map viewer for .hic files, written in JavaScript
                    and CSS. It is based on the desktop Juicebox visualization application, and
                    developed by the same team.</p>
                </div>
                <div style="text-align: center;" class="col-sm-6">
                    <img src="../img/juicebox-soft2.jpg" alt="juicebox" width="100%">(2)
                </div>
            </div>

            <div style="background: white; padding: 30px;" class="row" id="section6">
                <div style="text-align: center;" class="col-sm-6">
                    <img src="../img/gsea-soft.gif" alt="gsea" width="100%">(3)
                </div>
                <div id="section2" class="col-sm-6">
                    <h2><a target="_blank" href="http://software.broadinstitute.org/gsea/index.jsp"><img src="../img/software/GSEA-logo.gif" width="100px;">
                            GSEA</a>
                    </h2>
                    <hr>
                    <p>Gene Set Enrichment Analysis (GSEA) is a computational method that determines whether an
                    a priori defined set of genes shows statistically significant, concordant differences
                    between two biological states (e.g. phenotypes).</p>
                </div>
            </div>

            <div style="background: white; padding: 30px;" class="row" id="section7">
                <div id="section2" class="col-sm-6">
                    <h2><a target="_blank" href="http://msigdb.org"><img src="../img/software/MSigDB-logo-small.gif" width="50px;">
                            MSigDB</a>
                    </h2>
                    <hr>
                    <p>The Molecular Signatures Database (MSigDB) is a collection of annotated gene sets for
                    use with GSEA software. From this website, you can search for gene sets by keyword,
                    browse gene sets by name or collection, examine a gene set and its annotations,
                    download gene sets, compute overlaps between your gene set and gene sets in MSigDB,
                    Categorize members of a gene set by gene families, and view the expression profile of a
                    gene set in a provided public expression compendia.</p>
                </div>
                <div style="text-align: center;" class="col-sm-6">
                    <img src="../img/msigdb-soft.png" alt="msigdb" width="100%">
                </div>
            </div>

            <div style="background: white; padding: 30px;" class="row" id="section8">
                <div style="text-align: center;" class="col-sm-6">
                    <img src="/img/software/medullo-ss.png" width="100%">
                </div>
                <div id="section2" class="col-sm-6">
                    <h2><a href="/medulloblastoma/"> Medulloblastoma Portal</a>
                    </h2>
                    <hr>
                    <p>Medulloblastoma is the most common malignant brain cancer in children. Seventy percent of patients survive, but only 10% are able to live independently as adults due to neurologic disabilities from the tumor and treatment - specifically radiation. Understanding the disease can rapidly improve the quality of the life in survivors by more accurately identifying high and low risk tumors. Towards that goal, we and others are working to identify the drivers of medulloblastoma, such as mutations or chromosome alterations, which will enable development of more effective and less toxic targeted therapies. This portal contains data resources and supplementary files from our medulloblastoma publications as well as links to other relevant portals and resources. 
                    </p>
                </div>     
            </div>
            <div>
                <ol style="padding: 30px; margin-left:50px;">
                    <h2>References</h2>
                    <li>Helga Thorvaldsdóttir, James T. Robinson, Jill P. Mesirov. Integrative Genomics Viewer (IGV): high-performance
                        genomics
                        data visualization and exploration. Briefings in Bioinformatics 14, 178-192 (2013).</li>
                    <li>James T. Robinson, Douglass Turner, Neva C. Durand, Helga Thorvaldsdóttir, Jill P. Mesirov, Erez Lieberman
                        Aiden.
                        Juicebox.js Provides a Cloud-Based Visualization System for Hi-C Data. Cell Systems (2018).</li>
                    <li>http:/../software.broadinstitute.org/gsea/index.jsp</li>
                </ol>
            </div>
        </div>

        <div class="col-sm-2" style="padding:0px; margin:auto; align-content: right;">
            <nav id="myScrollspy" style="padding: 0px; width:100%">
                <ul class="nav nav-pills nav-stacked" style="margin-top:20px;">
                    <h1 style="text-align: left; padding:10px; margin: auto; font-size: 20px; background: #DEE7EF;font-weight: 100;">Software Tools</h1>
                    <li><a href="#title" style="font-size: 17px;">GenePattern</a></li>
                    <li><a href="#section1" style="font-size: 17px;">GenePattern Notebooks</a></li>
                    <li><a href="#section2" style="padding-right: 200px; font-size: 17px">IGV</a></li>
                    <li><a href="#section3" style="font-size: 17px;">igv.js</a></li>
                    <li><a href="#section4" style="font-size: 17px;">Juicebox.js</a></li>
                    <li><a href="#section5" style="font-size: 17px;">GSEA</a></li>
                    <li><a href="#section6" style="font-size: 17px;">MSigDB</a></li>
                    <li><a href="#section7" style="padding-right: 50px; font-size: 17px;">Medulloblastoma Portal</a></li>
                </ul>
            </nav>
        </div>
    </div>

            <!-- <div  style="background: white;padding: 50px;">
                <div class="panel-group" id="accordion">
                    <div class="panel panel-default" id="section1">
                        <div class="panel-heading row" data-toggle="collapse" data-parent="#accordion" href="#collapse1" style="padding-bottom: 0px;">
                            <div class="container-fluid col-sm-10" style="padding: 31px; padding-bottom: 0px;">
                                <img src="../img/software/genepattern.png">
                                <h2 class="panel-title">GenePattern &amp; GenePattern Notebook
                                </h2>
                            </div>
                            <div class="col-sm-2" style="padding: 31px; text-align: right">
                                <div class="glyphicon glyphicon-chevron-right" style="font-size: 20px"></div>
                            </div>
                        </div>
                        <div id="collapse1" class="panel-collapse collapse">
                            <div class="panel-body" style="padding: 3%;">
                                <h2><a target="_blank" href="https://genepattern.broadinstitute.org">GenePattern</a></h2>
                                <p>We have developed a software package called GenePattern, which provides a comprehensive environment that can support a broad community of users at all levels of computational experience and sophistication, access to a repository of analytic and visualization tools and easy creation of complex analytic methods from them and the rapid development and dissemination of new methods. Perhaps the most important feature of GenePattern is that it supports a mechanism to guarantee the capture and independent replication of published computational methods and in silico results.</p>
                                <ol>
                                    <li>
                                        Reich M, Liefeld T, Gould J, Lerner J,
                                        Tamayo P, Mesirov JP. <b>GenePattern 2.0</b>. Nature Genetics 38 no. 5 (2006): pp500-501
                                    </li>
                                    <li>
                                        Tan Y, Wu F, Tamayo P, Haining WN, Mesirov JP. <b>Constellation Map: Downstream
                                        visualization and interpretation of gene set enrichment results.</b> F1000Res. 2015
                                        Jun 24;4:167. doi: 10.12688/f1000research.6644.1. eCollection 2015. PubMed PMID:
                                        26594333; PubMed Central PMCID: PMC4648224.
                                    </li>
                                    <li>
                                        Spidlen J, Barsky A, Breuer K, Carr P, Nazaire MD, Hill BA, Qian Y, Liefeld T,
                                        Reich M, Mesirov JP, Wilkinson P, Scheuermann RH, Sekaly RP, Brinkman RR.
                                        <b>GenePattern flow cytometry suite. </b>Source Code Biol Med. 2013 Jul 3;8(1):14. doi:
                                        10.1186/1751-0473-8-14. PubMed PMID: 23822732; PubMed Central PMCID: PMC3717030.
                                    </li>
                                    <li>
                                        Kuehn H, Liberzon A, Reich M, Mesirov JP. <b>Using GenePattern for gene
                                        expression analysis.</b> Curr Protoc Bioinformatics. 2008 Jun;Chapter 7:Unit 7.12.
                                        doi: 10.1002/0471250953.bi0712s22. Review. PubMed PMID: 18551415; PubMed Central
                                        PMCID: PMC3893799.
                                    </li>
                                </ol><br>
                                <h2><a target="_blank" href="https://genepattern-notebook.org/">GenePattern Notebook</a></h2>
                                <p>Interactive analysis notebook environments promise to streamline genomics research
                                    through interleaving text, multimedia, and executable code into unified, sharable,
                                    reproducible “research narratives.” However, current notebook systems require
                                    programming knowledge, limiting their wider adoption by the research community. We have
                                    developed the GenePattern Notebook environment, to our knowledge the first system to
                                    integrate the dynamic capabilities of notebook systems with an investigator-focused,
                                    easy-to-use interface that provides access to hundreds of genomic tools without the
                                    need to write code.</p>
                                <ol>
                                    <li>Reich et al. <b>The GenePattern
                                        Notebook Environment</b>. Cell Systems. 2017(5):149-51</li>
                                    </ol>
                            </div>
                        </div>
                    </div>
        
                    <div class="panel panel-default" id="section2">
                        <div class="panel-heading row" data-toggle="collapse" data-parent="#accordion" href="#collapse2" style="padding-bottom: 0px;">
                            <div class="container-fluid col-sm-10" style="padding: 31px; padding-bottom: 0px;">
                                <img src="../img/software/11527111.png">
                                <h2 class="panel-title">The Integrative Genomics Viewer (IGV)
                                </h2>
                            </div>
                            <div class="col-sm-2" style="padding: 31px; text-align: right">
                                <div class="glyphicon glyphicon-chevron-right" style="font-size: 20px"></div>
                            </div>
                        </div>
                        <div id="collapse2" class="panel-collapse collapse">
                            <div class="panel-body" style="padding: 3%;">
                                <h2><a target="_blank" href="https://igv.org/">IGV</a></h2>
                                <p>The Integrative Genomics Viewer (IGV) is a high-performance visualization tool for
                                    interactive exploration of large, integrated genomic datasets. It supports a wide
                                    variety of data types, including array-based and next-generation sequence data, and
                                    genomic annotations.</p>
                                <ol>
                                    <li>
                                        Robinson JT, Thorvaldsdottir H, Mesirov J. <b>Exploring cancer datasets in the integrative genomics viewer (IGV). </b>Cancer
                                        Research. 2012;72. doi: 10.1158/1538-7445.am2012-3968.
                                    </li>
                                    <li>
                                        James T. Robinson, Helga
                                        Thorvaldsdóttir, Wendy Winckler, Mitchell Guttman, Eric S. Lander, Gad Getz, Jill P.
                                        Mesirov. <b>Integrative Genomics Viewer.</b> Nature Biotechnology 29, 24–26 (2011)
                                    </li>
                                    <li>
                                        Helga Thorvaldsdóttir, James T. Robinson, Jill P. Mesirov. <b>Integrative Genomics
                                            Viewer (IGV): high-performance genomics data visualization and exploration.</b>
                                        Briefings in Bioinformatics 14, 178-192 (2013).
                                    </li>
                                    <li>
                                        Robinson JT, Thorvaldsdóttir H, Wenger AM, Zehir A, Mesirov JP. <b>Variant Review
                                        with the Integrative Genomics Viewer.</b> Cancer Res. 2017 Nov 1;77(21):e31-e34. doi:
                                        10.1158/0008-5472.CAN-17-0337. PubMed PMID: 29092934; PubMed Central PMCID:
                                        PMC5678989.
                                    </li>
                                </ol><br>
                                <h2><a target="_blank" href="https://igv.org/doc/doc.html">IGV.js</a></h2>
                                <p>IGV.js is an embeddable interactive genome visualization component written in JavaScript and CSS. It is based on the Integrative Genomics Viewer(IGV), and developed by the same team.</p><br>
                                <h2><a target="_blank" href="https://igv.org/doc/juiceboxjs.html">Juicebox.js</a></h2>
                                Juicebox.js is an interactive contact map viewer for .hic files, written in JavaScript
                                and CSS. It is based on the desktop Juicebox visualization application, and
                                developed by the same team.
                                <ol>
                                    <li>
                                        Robinson JT, Turner D, Durand NC,
                                        Thorvaldsdóttir H, Mesirov JP, Aiden EL. <b>Juicebox.js Provides a Cloud-Based
                                            Visualization System for Hi-C Data.</b> Cell Syst. 2018 Feb 28;6(2):256-258.e1.
                                        doi: 10.1016/j.cels.2018.01.001. Epub 2018 Feb 7. PubMed PMID: 29428417; PubMed Central
                                        PMCID: PMC6047755.
                                    </li>
                                </ol>
                            </div>
                        </div>
                    </div>
        
                    <div class="panel panel-default" id="section3">
        
                        <div class="panel-heading row" data-toggle="collapse" data-parent="#accordion" href="#collapse3" style="padding-bottom: 0px;">
                            <div class="container-fluid col-sm-10" style="padding: 31px; padding-bottom: 10px;">
                                <img src="../img/software/gsea-msigdb.jpg" id="gsea-header">
                                <h2 class="panel-title title" style="margin-top:-10px;">Gene Set Enrichment Analysis &amp; Molecular Signatures Database
                                </h2>
                            </div>
                            <div class="col-sm-2" style="padding: 31px; text-align: right">
                                <div class="glyphicon glyphicon-chevron-right" style="font-size: 20px"></div>
                            </div>
                        </div>
                        <div id="collapse3" class="panel-collapse collapse">
                            <div class="panel-body" style="padding: 3%;">
                                <h2><a target="_blank" href="https:/../software.broadinstitute.org/gsea/index.jsp">GSEA</a></h2>
                                <p>Gene Set Enrichment Analysis (GSEA) is a computational method that determines whether an
                                    a priori defined set of genes shows statistically significant, concordant differences
                                    between two biological states (e.g. phenotypes).</p>
                                <ol>
                                <li>Subramanian A, Tamayo P, Mootha VK,
                                    Mukherjee S, Ebert BL, Gillette MA, Paulovich A, Pomeroy SL, Golub TR, Lander ES,
                                    Mesirov JP. <b>Gene set enrichment analysis: a knowledge-based approach for
                                        interpreting genome-wide expression profiles.</b> Proc Natl Acad Sci U S A. 2005
                                    Oct 25;102(43):15545-50. Epub 2005 Sep 30. PubMed PMID: 16199517; PubMed Central PMCID:
                                    PMC1239896.
                                </li>
                                <li>Mootha VK, Lindgren CM, Eriksson KF,
                                    Subramanian A, Sihag S, Lehar J, Puigserver P, Carlsson E, Ridderstråle M, Laurila E,
                                    Houstis N, Daly MJ, Patterson N, Mesirov JP, Golub TR, Tamayo P, Spiegelman B, Lander
                                    ES, Hirschhorn JN, Altshuler D, Groop LC. <b>PGC-1alpha-responsive genes involved in
                                        oxidative phosphorylation are coordinately downregulated in human diabetes.</b> Nat
                                    Genet. 2003 Jul;34(3):267-73. PubMed PMID: 12808457.
                                </li>
                                <li>
                                    Subramanian A, Kuehn H, Gould J, Tamayo P, Mesirov JP. <b>GSEA-P: a desktop
                                    application for Gene Set Enrichment Analysis. </b>Bioinformatics. 2007 Dec
                                    1;23(23):3251-3. Epub 2007 Jul 20. PubMed PMID: 17644558.
                                </li>
                                </ol>
                                    <br>
                                <h2><a target="_blank" href="https:/../software.broadinstitute.org/gsea/index.jsp">MSigDB</a></h2>
                                <p>The Molecular Signatures Database (MSigDB) is a collection of annotated gene sets for
                                    use with GSEA software. From this website, you can search for gene sets by keyword,
                                    browse gene sets by name or collection, examine a gene set and its annotations,
                                    download gene sets, compute overlaps between your gene set and gene sets in MSigDB,
                                    Categorize members of a gene set by gene families, and view the expression profile of a
                                    gene set in a provided public expression compendia.
                                </p>
                                <ol>
                                    <li>Liberzon A, Subramanian A, Pinchback R, Thorvaldsdóttir H, Tamayo P, Mesirov
                                    JP. <b>Molecular signatures database (MSigDB) 3.0</b>. Bioinformatics. 2011 Jun
                                    15;27(12):1739-40. doi: 10.1093/bioinformatics/btr260. Epub 2011 May 5. PubMed
                                    PMID: 21546393; PubMed Central PMCID: PMC3106198.
                                    </li>
                                    <li>
                                        Arthur Liberzon, Chet Birger, Helga
                                        Thorvaldsdóttir, Mahmoud Ghandi, Jill P. Mesirov, Pablo Tamayo, <b>The Molecular
                                        Signatures Database Hallmark Gene Set Collection</b>, In Cell Systems, Volume 1,
                                        Issue 6, 2015, Pages 417-425, ISSN 2405-4712,
                                        https://doi.org/10.1016/j.cels.2015.12.004.
                                    </li>
                                </ol>
                                
                            </div>
                        </div>
                    </div>
        
                    <div class="panel panel-default" id="section4">
                        <div class="panel-heading row" data-toggle="collapse" data-parent="#accordion" href="#collapse4" style="padding-bottom: 0px;">
                            <div class="container-fluid col-sm-10" style="padding: 31px; padding-bottom: 0px;">
                                <img src="../img/software/gs.png">
                                <h2 class="panel-title">GenomeSpace
                                </h2>
                            </div>
                            <div class="col-sm-2" style="padding: 31px; text-align: right">
                                <div class="glyphicon glyphicon-chevron-right" style="font-size: 20px"></div>
                            </div>
                        </div>
                        <div id="collapse4" class="panel-collapse collapse">
                            <div class="panel-body" style="padding: 3%;">
                                <h2><a target="_blank" href="https://www.genomespace.org/">GenomeSpace</a></h2>
                                <p>GenomeSpace is a cloud-based interoperability framework to support integrative genomics
                                    analysis through an easy-to-use Web interface. GenomeSpace provides access to a diverse
                                    range of bioinformatics tools, and bridges the gaps between the tools, making it easy
                                    to leverage the available analyses and visualizations in each of them. The tools retain
                                    their native look and feel, with GenomeSpace providing frictionless conduits between
                                    them through a lightweight interoperability layer. GenomeSpace does not perform any
                                    analyses itself; these are done within the member tools wherever they live – desktop,
                                    Web service, cloud, in-house server, etc. Rather, GenomeSpace provides tool selection
                                    and launch capabilities, and acts as a data highway automatically reformatting data as
                                    required when results move from the output of one tool to input for the next.
                                </p>
                                <ol>
                                    <li>
                                        Qu K, Garamszegi S, Wu F, et al. <b>Integrative
                                            genomic analysis by interoperation of bioinformatics tools in GenomeSpace.</b>
                                        Nature methods. 2016;13(3):245-247. doi:10.1038/nmeth.3732.
                                    </li>
                                    <li>
                                        Carlin D, Kosnicki K, Garamszegi S, Ideker T, Thorvaldsdóttir H, Reich M,
                                        Mesirov J. <b>A multi-tool recipe to identify regions of protein-DNA binding and
                                        their influence on associated gene expression.</b> Version 2. F1000Res. 2017 Jun 6
                                        [revised 2018 Jan 1];6:784. doi: 10.12688/f1000research.11616.2. eCollection
                                        2017. PubMed PMID: 29487738; PubMed Central PMCID: PMC5806050.

                                    </li>
                                    <li>
                                        Demchak B, Hull T, Reich M, Liefeld T, Smoot M, Ideker T, Mesirov JP.
                                        <b>Cytoscape: the network visualization tool for GenomeSpace workflows.</b> Version 2.
                                        F1000Res. 2014 Jul 1 [revised 2014 Jan 1];3:151. doi:
                                        10.12688/f1000research.4492.2. eCollection 2014. PubMed PMID: 25165537; PubMed
                                        Central PMCID: PMC4133763.

                                    </li>
                                </ol>
                                
                            </div>
                        </div>
                    </div>
        
                </div>
            </div>
        </div> -->

    <!-- <div id="software-main">
        <div class="container" id="software-container" style="padding: 30px;">
            
            <div id="title" style="padding: 30px;">
                <h1>SOFTWARE</h1>
            </div>
            
            <div style="padding: 50px; margin-top: -30px; margin-bottom: -30px;">
                <div class="panel-group" id="accordion">
                    <div class="panel panel-default">
                        
                        <div class="panel-heading row" data-toggle="collapse" data-parent="#accordion" href="#collapse1" style="padding-bottom: 0px;">
                            <div class="container-fluid col-sm-10" style="padding: 31px; padding-bottom: 0px;">
                                <img src="../img/software/genepattern.png">
                                <h2 class="panel-title">GenePattern &amp; GenePattern Notebook
                                </h2>
                            </div>
                            <div class="col-sm-2" style="padding: 31px; text-align: right">
                                <div class="glyphicon glyphicon-chevron-right" style="font-size: 20px"></div>
                            </div>
                        </div>
                        <div id="collapse1" class="panel-collapse collapse">
                            <div class="panel-body" style="padding: 3%;">
                                <h2><a target="_blank" href="https://genepattern.broadinstitute.org">GenePattern</a></h2>
                                <p>We have developed a software package called GenePattern, which provides a comprehensive environment that can support a broad community of users at all levels of computational experience and sophistication, access to a repository of analytic and visualization tools and easy creation of complex analytic methods from them and the rapid development and dissemination of new methods. Perhaps the most important feature of GenePattern is that it supports a mechanism to guarantee the capture and independent replication of published computational methods and in silico results.</p>
                                <p style="color: rgba(0,0,0,0.6); font-size: 14px;">Reich M, Liefeld T, Gould J, Lerner J, Tamayo P, Mesirov JP. <b>GenePattern 2.0</b>. Nature Genetics 38 no. 5 (2006): pp500-501</p>
                                <h2><a target="_blank" href="https://genepattern-notebook.org/">GenePattern Notebook</a></h2>
                                <p>Interactive analysis notebook environments promise to streamline genomics research through interleaving text, multimedia, and executable code into unified, sharable, reproducible “research narratives.” However, current notebook systems require programming knowledge, limiting their wider adoption by the research community. We have developed the GenePattern Notebook environment, to our knowledge the first system to integrate the dynamic capabilities of notebook systems with an investigator-focused, easy-to-use interface that provides access to hundreds of genomic tools without the need to write code.</p>
                                <p style="color: rgba(0,0,0,0.6); font-size: 14px;">Reich et al. <b>The GenePattern Notebook Environment</b>. Cell Systems. 2017(5):149-51</p>
                            </div>
                        </div>
                    </div>
                
                    <div class="panel panel-default">
                        <div class="panel-heading row" data-toggle="collapse" data-parent="#accordion" href="#collapse2" style="padding-bottom: 0px;">
                            <div class="container-fluid col-sm-10" style="padding: 31px; padding-bottom: 0px;">
                                <img src="../img/software/11527111.png"> 
                                <h2 class="panel-title">The Integrative Genomics Viewer (IGV)
                                </h2>
                            </div>
                            <div class="col-sm-2" style="padding: 31px; text-align: right">
                                <div class="glyphicon glyphicon-chevron-right" style="font-size: 20px"></div>
                            </div>
                        </div>
                        <div id="collapse2" class="panel-collapse collapse">
                            <div class="panel-body" style="padding: 3%;">
                                <h2><a target="_blank" href="https://igv.org/">IGV</a></h2>
                                <p>The Integrative Genomics Viewer (IGV) is a high-performance visualization tool for interactive exploration of large, integrated genomic datasets. It supports a wide variety of data types, including array-based and next-generation sequence data, and genomic annotations.</p>
                                <p style="color: rgba(0,0,0,0.6); font-size: 14px;">James T. Robinson, Helga Thorvaldsdóttir, Wendy Winckler, Mitchell Guttman, Eric S. Lander, Gad Getz, Jill P. Mesirov. <b>Integrative Genomics Viewer.</b> Nature Biotechnology 29, 24–26 (2011)</p>
                                <p style="color: rgba(0,0,0,0.6); font-size: 14px;">
                                Helga Thorvaldsdóttir, James T. Robinson, Jill P. Mesirov. <b>Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration.</b> Briefings in Bioinformatics 14, 178-192 (2013).</p>
                                <h2><a target="_blank" href="https://igv.org/doc/doc.html">IGV.js</a></h2>
                                <p>IGV.js is an embeddable interactive genome visualization component written in JavaScript and CSS. It is based on the Integrative Genomics Viewer(IGV), and developed by the same team.</p>
                                <h2><a target="_blank" href="https://igv.org/doc/juiceboxjs.html">Juicebox.js</a></h2>
                                <p>Juicebox.js is an interactive contact map viewer for .hic files, written in JavaScript and CSS. <br>It is based on the desktop Juicebox visualization application, and developed by the same team.</p>
                                <p style="color: rgba(0,0,0,0.6); font-size: 14px;">Robinson JT, Turner D, Durand NC, Thorvaldsdóttir H, Mesirov JP, Aiden EL. <b>Juicebox.js Provides a Cloud-Based Visualization System for Hi-C Data.</b> Cell Syst. 2018 Feb 28;6(2):256-258.e1. doi: 10.1016/j.cels.2018.01.001. Epub 2018 Feb 7. PubMed PMID: 29428417; PubMed Central PMCID: PMC6047755.</p>
                            </div>
                        </div>
                    </div>
                    
                    <div class="panel panel-default">
                        
                        <div class="panel-heading row" data-toggle="collapse" data-parent="#accordion" href="#collapse3" style="padding-bottom: 0px;">
                           <div class="container-fluid col-sm-10" style="padding: 31px; padding-bottom: 0px;">
                                <img src="../img/software/gsea-msigdb.jpg" id="gsea-header">
                                <h2 class="panel-title">Gene Set Enrichment Analysis &amp; Molecular Signatures Database
                                </h2>
                            </div>
                            <div class="col-sm-2" style="padding: 31px; text-align: right">
                                <div class="glyphicon glyphicon-chevron-right" style="font-size: 20px"></div>
                            </div>
                        </div>
                        <div id="collapse3" class="panel-collapse collapse">
                            <div class="panel-body" style="padding: 3%;">
                                <h2><a target="_blank" href="https:/../software.broadinstitute.org/gsea/index.jsp">GSEA</a></h2>
                                <p>Gene Set Enrichment Analysis (GSEA) is a computational method that determines whether an a priori defined set of genes shows statistically significant, concordant differences between two biological states (e.g. phenotypes).</p>
                                <p style="color: rgba(0,0,0,0.6); font-size: 14px;">Subramanian A, Tamayo P, Mootha VK, Mukherjee S, Ebert BL, Gillette MA, Paulovich A, Pomeroy SL, Golub TR, Lander ES, Mesirov JP. <b>Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles.</b> Proc Natl Acad Sci U S A. 2005 Oct 25;102(43):15545-50. Epub 2005 Sep 30. PubMed PMID: 16199517; PubMed Central PMCID: PMC1239896.</p>
                                <p style="color: rgba(0,0,0,0.6); font-size: 14px;">Mootha VK, Lindgren CM, Eriksson KF, Subramanian A, Sihag S, Lehar J, Puigserver P, Carlsson E, Ridderstråle M, Laurila E, Houstis N, Daly MJ, Patterson N, Mesirov JP, Golub TR, Tamayo P, Spiegelman B, Lander ES, Hirschhorn JN, Altshuler D, Groop LC. <b>PGC-1alpha-responsive genes involved in oxidative phosphorylation are coordinately downregulated in human diabetes.</b> Nat Genet. 2003 Jul;34(3):267-73. PubMed PMID: 12808457.</p>
                                <h2><a target="_blank" href="https:/../software.broadinstitute.org/gsea/index.jsp">MSigDB</a></h2>
                                <p>The Molecular Signatures Database (MSigDB) is a collection of annotated gene sets for use with GSEA software. From this website, you can search for gene sets by keyword, browse gene sets by name or collection, examine a gene set and its annotations, download gene sets, compute overlaps between your gene set and gene sets in MSigDB, Categorize members of a gene set by gene families, and view the expression profile of a gene set in a provided public expression compendia.
                                </p>
                                <p style="color: rgba(0,0,0,0.6); font-size: 14px;">Arthur Liberzon, Chet Birger, Helga Thorvaldsdóttir, Mahmoud Ghandi, Jill P. Mesirov, Pablo Tamayo, <b>The Molecular Signatures Database Hallmark Gene Set Collection</b>, In Cell Systems, Volume 1, Issue 6, 2015, Pages 417-425, ISSN 2405-4712, https://doi.org/10.1016/j.cels.2015.12.004.</p>
                            </div>
                        </div>
                    </div>
                    
                    <div class="panel panel-default">
                        <div class="panel-heading row" data-toggle="collapse" data-parent="#accordion" href="#collapse4" style="padding-bottom: 0px;">
                            <div class="container-fluid col-sm-10" style="padding: 31px; padding-bottom: 0px;">
                                <img src="../img/software/gs.png">
                                <h2 class="panel-title">GenomeSpace
                                </h2>
                            </div>
                            <div class="col-sm-2" style="padding: 31px; text-align: right">
                                <div class="glyphicon glyphicon-chevron-right" style="font-size: 20px"></div>
                            </div>
                        </div>
                        <div id="collapse4" class="panel-collapse collapse">
                            <div class="panel-body" style="padding: 3%;">
                                <h2><a target="_blank" href="https://www.genomespace.org/">GenomeSpace</a></h2>
                                <p>GenomeSpace is a cloud-based interoperability framework to support integrative genomics analysis through an easy-to-use Web interface. GenomeSpace provides access to a diverse range of bioinformatics tools, and bridges the gaps between the tools, making it easy to leverage the available analyses and visualizations in each of them. The tools retain their native look and feel, with GenomeSpace providing frictionless conduits between them through a lightweight interoperability layer. GenomeSpace does not perform any analyses itself; these are done within the member tools wherever they live – desktop, Web service, cloud, in-house server, etc. Rather, GenomeSpace provides tool selection and launch capabilities, and acts as a data highway automatically reformatting data as required when results move from the output of one tool to input for the next.
                                </p>
                                <p style="color: rgba(0,0,0,0.6); font-size: 14px;">Qu K, Garamszegi S, Wu F, et al. <b>Integrative genomic analysis by interoperation of bioinformatics tools in GenomeSpace.</b> Nature methods. 2016;13(3):245-247. doi:10.1038/nmeth.3732.</p>
                            </div>
                        </div>
                    </div>
                    
                </div>
            </div>
        </div>
    </div> -->
    
    <footer style="margin-top: 0px; padding: 50px; padding-bottom: 80px; ">
            <div class="row">
                <div class="col-sm-3" style="padding-top:0px; ">
                    <h1 style="margin-top:0px;">Mesirov Lab</h1>
                    <p>
                        Department of Medicine<br>
                        UC San Diego, School of Medicine<br>
                        Floor 2A, Biomedical Research Facility II<br>
                        La Jolla, CA 92093<br><br>
    
                        
                        
    
                        Mail Code: 0602<br>
                    </p>
                    <p style="font-size: 10px;">© Copyright Mesirov Lab 2018 </p>
                </div>
                <div class="col-sm-4">   
                </div>
                <div class="col-sm-5 row">
                    <div class="col-sm-4">
                        <a href="/members/">Our Team</a><br><br>
                        <a href="/projects/">Research</a>
                    </div>
                    <div class="col-sm-4">
                        <a href="/pubs/">Publications</a><br><br>
                        <a href="/software/">Software</a>
                    </div>
                    <div class="col-sm-4">
                        <a href="/teaching/">Teaching</a><br><br>
                        <a href="/about/">Contact</a><br><br>
                        <a href="/opportunities/">Join Us</a>
                    </div>
                    <div class="col-12" style="text-align: right;">
                        
                    </div>
                </div>
            </div>
            <!-- <p>Mesirov Lab | UC San Diego School of Medicine | 9500 Gilman Drive | La Jolla, CA 92093</p> -->
        </footer>
    
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
    <script src="../js/smooth-scroll.js"></script>
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
</body>  
</html>
