<!DOCTYPE html>

<html 
      xmlns="http://www.w3.org/1999/xhtml" 
      xml:lang="en" 
      version="XHTML+RDFa 1.0" 
      dir="ltr" 
      xmlns:content="http://purl.org/rss/1.0/modules/content/"
      xmlns:dc="http://purl.org/dc/terms/" 
      xmlns:foaf="http://xmlns.com/foaf/0.1/" 
      xmlns:og="http://ogp.me/ns#" 
      xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" 
      xmlns:sioc="http://rdfs.org/sioc/ns#" 
      xmlns:sioct="http://rdfs.org/sioc/types#" 
      xmlns:skos="http://www.w3.org/2004/02/skos/core#" 
      xmlns:xsd="http://www.w3.org/2001/XMLSchema#" 
      class="js">

<!-- Head/Meta Data -->
<head>
    <title>Archer 2018 | Medulloblastoma</title>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
    <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet" type="text/css">
    <link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet" type="text/css">
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
    <script src="../medulloblastoma_files/medulloblastoma.js"></script>
    <link href="../medulloblastoma_files/pubs.css" rel="stylesheet" type="text/css">
</head>

<!-- Body/Content -->
<body id="myPage" data-spy="scroll" data-target=".navbar" data-offset="100">

<!-- Navigation Bar -->
<nav class="navbar navbar-default navbar-fixed-top">
    <div class="navbar-header">
        <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#myNavbar">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>                        
        </button>
        <a class="navbar-brand" href="..">Medulloblastoma Portal</a>
    </div>
    <div class="collapse navbar-collapse" id="myNavbar">
        <ul class="nav navbar-nav navbar-right">
            <li><a href="..">Home</a></li>
            <li><a href="../about/">Contact Us</a></li>
        </ul>
    </div>
</nav>

<!-- Title Container -->
<div id="title">
    <div class="container-fluid text-center">
        <h1>
            Proteomics, Post-translational Modifications, and Integrative Analyses Reveal Molecular Heterogeneity within Medulloblastoma Subgroups
        </h1>
        <hr>
        <br>
        <p>
            Archer TC, Ehrenberger T, Mundt F, Gold MP, Krug K, Mah CK, Mahoney EL, Daniel
            CJ, LeNail A, Ramamoorthy D, Mertins P, Mani DR, Zhang H, Gillette MA, Clauser K,
            Noble M, Tang LC, Pierre-François J, Silterra J, Jensen J, Tamayo P, Korshunov A,
            Pfister SM, Kool M, Northcott PA, Sears RC, Lipton JO, Carr SA, Mesirov JP,
            Pomeroy SL, Fraenkel E. Proteomics, Post-translational Modifications, and
            Integrative Analyses Reveal Molecular Heterogeneity within Medulloblastoma
            Subgroups. Cancer Cell. 2018 Sep 10;34(3):396-410.e8. doi:
            10.1016/j.ccell.2018.08.004. PubMed PMID: 30205044.
        </p>
    </div>
</div>

<!-- Abstract Container -->
<div id="abstract">
        <div class="container-fluid">
            <div class="col-12 col-sm-7">
                <h3>ABSTRACT</h3>
                <p>There is a pressing need to identify therapeutic targets in tumors with low mutation rates such as the malignant pediatric brain tumor medulloblastoma. To address this challenge, we quantitatively profiled global proteomes and phospho-proteomes of 45 medulloblastoma samples. Integrated analyses revealed that tumors
                with similar RNA expression vary extensively at the post-transcriptional and post-translational levels. We
                identified distinct pathways associated with two subsets of SHH tumors, and found post-translational modifications of MYC that are associated with poor outcomes in group 3 tumors. We found kinases associated
                with subtypes and showed that inhibiting PRKDC sensitizes MYC-driven cells to radiation. Our study shows
                that proteomics enables a more comprehensive, functional readout, providing a foundation for future therapeutic
                strategies.</p>
            </div>
            <div class="col-12 col-sm-5" style="margin:auto;"> 
                <img src="archer_files/abstract.png" height="400px;">
            </div>
            <hr style="margin-top: 500px;">
            <h5>
                Highlights
            </h5>
            <ul>
                <li>Deep proteomic profiling reveals mechanistic differences among medulloblastomas
                </li>
                <li>Proteomic-defined SHH subtypes do not differ in RNA expression</li>
                <li>MYC phosphorylation events define a higher risk subset of group 3 patients</li>
                <li>Inhibiting PRKDC may sensitize MYC-activated medulloblastoma tumors to radiation</li>
            </ul>
            <br>
            <h5>
               Significance
            </h5>
            <p>
                Genomic and epigenomic analyses have revolutionized cancer diagnostics. Nevertheless, it has been difficult to identify therapeutic targets for tumors that lack recurrent genomic lesions. Here we used global, mass spectrometry-based measurements of protein levels and post-translational modifications to identify functional pathways associated with subtypes of medulloblastoma. Strong proteomic signals revealed altered pathways that were not detected transcriptionally. One molecular subgroup of tumors showed marked discordance of RNA and protein levels, suggesting global changes in translation and/or proteostasis. We demonstrate the utility of an integrative approach for discovery of candidate biomarkers or drug targets and provide a multi-omic dataset that will serve as a resource for the community. This study has the potential to impact clinical trial design.
            </p>
            <br>
            <h5>Data & Availability</h5>
            <p>
                <li>The accession number for the proteomic, phospho-proteomic, and acetylome data reported in this paper is <a href="https://gnps.ucsd.edu/ProteoSAFe/result.jsp?task=e8d5a5716e1a4ecf8e4653aa47d773a5&view=advanced_view">MSV000082644</a>
                </li>
                <li>Group 3 cohort expansion tool reported in the paper can be located at <a href="https://github.com/ckmah/archer2018_g3_cohort_expansion/">https://github.com/ckmah/archer2018_g3_cohort_expansion/</a>.

                </li>
                <li>The genomic data used in this paper is a subset of the Northcott
                    et al. 2017 dataset, which can be found at <a href="https://www.nature.com/articles/nature22973">https://www.nature.com/articles/nature22973</a>.

                </li> 

                <li>To browse and explore the data from this study go to <a href="http://fraenkel.mit.edu/archer2018">http://fraenkel.mit.edu/archer2018</a>
                </li>
            </p>
        </div> 
</div>

<!-- Supplemental Material Container -->
<div id="supplemental">
        <div class="container-fluid">
            <h2>SUPPLEMENTAL MATERIAL</h2>
            <hr>
            <div class="documents">
                <ul>
                    <li>
                        Cancer Cell Paper: <a href="https://www.cell.com/cancer-cell/fulltext/S1535-6108(18)30358-1">Archer et al. 2018</a>
                    </li>
                    <li>
                        Supplemental Figures: 
                        <a href="https://www.cell.com/cms/10.1016/j.ccell.2018.08.004/attachment/85add6d5-3181-4aef-b043-baa694f66fac/mmc1.pdf">Archer et al. 2018 Supplemental</a>
                    </li>
                    <li>Supplemental Tables</li>
                    <ul>
                            <li><a href="https://www.cell.com/cms/10.1016/j.ccell.2018.08.004/attachment/83214c89-f51f-450b-b375-bf1a0ac8421f/mmc2.xlsx">Table S1</a>: Overview of Samples Included in this study, their Subtypes, Clinical Details, and Data Types Available, related to Figures 1 and 2.</li>
                            <li><a href="https://www.cell.com/cms/10.1016/j.ccell.2018.08.004/attachment/663b1659-8a50-432c-af27-c24fff22756e/mmc3.xlsx">Table S2</a>: Normalized proteomic and RNA-seq data, as well as differential proteomic features, related to Figure 2.</li>
                            <li><a href="https://www.cell.com/cms/10.1016/j.ccell.2018.08.004/attachment/0402289e-d57a-4df0-903c-bd3a35a9fb9e/mmc4.xlsx">Table S3</a>: Pathways Associated with High Protein Levels in SHHa and SHHb, related to Figure 3.</li>
                            <li><a href="https://www.cell.com/cms/10.1016/j.ccell.2018.08.004/attachment/989864d7-dace-4d1c-ad1b-aeabf1c2e3b4/mmc5.xlsx">Table S4</a>: Genes Included in Figure 3B, related to Figure 3.</li>
                            <li><a href="https://www.cell.com/cms/10.1016/j.ccell.2018.08.004/attachment/d05e17b2-9233-40cd-839d-3b891a3f90d9/mmc6.xlsx">Table S5</a>: Complete list of all kinase-peptide and kinase-drug associations found in kinome analysis, related to Figure 5. </li>
                            <li><a href="https://www.cell.com/cms/10.1016/j.ccell.2018.08.004/attachment/f6da5d83-6c67-4206-af06-62d594534a15/mmc7.xlsx">Table S6</a>: Prizes Used as Input to Omics Integrator, related to the STAR Methods.</li>
                        </ul>
                    <li>Genomic Data is a subset of: <a href="https://www.ebi.ac.uk/ega/studies/EGAS00001001953">EGAS00001001953</a></li>
                    <ul>
                        <li>Originally published in: <a href="https://www.nature.com/articles/nature22973">Northcott et al. 2017</a></li>
                    </ul>
                    <li>To browse and explore the data from this study go to <a href="http://fraenkel.mit.edu/archer2018">http://fraenkel.mit.edu/archer2018</a>
                    </li>
                    <li>
                        Complete Proteomic Dataset (accesion number: MSV00008264): <a href="https://gnps.ucsd.edu/ProteoSAFe/result.jsp?task=e8d5a5716e1a4ecf8e4653aa47d773a5&view=advanced_view">GNPS MassIVE Dataset (237.58GB)</a>
                    </li>

                    <li>
                        Group 3 cohort expansion tool: <a href="https://github.com/ckmah/archer2018_g3_cohort_expansion/">Github Repository</a>
                    </li>
                    
                    

                </ul>
            </div>
        </div>
</div>
    
<!-- Footer -->
    <footer class="text-center">
        <a class="up-arrow" href="#myPage" data-toggle="tooltip" title="TO TOP">
             <span class="glyphicon glyphicon-chevron-up"></span>
        </a>
        <br>
    </footer>
    
</body>
</html>
